android/src/main/java/com/brentvatne/common/toolbox/CaptureUtil.kt

```diff
@@ -0,0 +1,92 @@
+package com.brentvatne.common.toolbox
+
+import android.content.ContentValues
+import android.graphics.Bitmap
+import android.net.Uri
+import android.os.Build
+import android.os.Handler
+import android.os.Looper
+import android.provider.MediaStore
+import android.view.PixelCopy
+import android.view.SurfaceView
+import android.view.TextureView
+import android.view.View
+import androidx.media3.common.util.Util
```

Comment:
This import should not be in common folder : import androidx.media3.common.util.Util
Any way to remove it ?